lib/src/attributes/variants/variants_short.utils.dart

```diff
@@ -12,7 +12,16 @@ final landscape = VariantUtils.landscape();
 final dark = VariantUtils.dark();
 final light = VariantUtils.light();
 
+final enabled = not(disabled);
 final disabled = VariantUtils.disabled();
 final focus = VariantUtils.focused();
 final hover = VariantUtils.hover();
 final press = VariantUtils.pressing();
+
+final active = VariantUtils.active();
```

Comment:
What is an active variant also. Primitives don't have the concept of active, maybe this is part of the headless conversation